zone/lua_parser_events.cpp

```diff
@@ -330,6 +330,9 @@ void handle_npc_death(
 		l_npc_o.push(L);
 		lua_setfield(L, -2, "killed");
 	}
+
```

Comment:
Does there become a weird ordering behavior with extra parameters and the extra pointers ?

Reply:
Shouldn't be any issue since we're exporting them named and the order shouldn't matter much.

I may be able to just remove the `GetID()` export altogether since we now have it functioning properly and don't need this in Lua since we'll have access to the pointer.